.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
+  "trailingComma": "es5",
+  "tabWidth": 2,
+  "semi": true,
+  "doubleQuote": true,
+  "printWidth": 90
+}
```

Comment:
I love this, great job. @brobuck87 make sure you have Prettier set as your default formatter and it will listen to this configuration file.